src/fiatconnect/FiatDetailsScreen.test.tsx

```diff
@@ -130,9 +130,49 @@ describe('FiatDetailsScreen', () => {
 
     expect(queryByTestId('errorMessage')).toBeFalsy()
 
-    expect(queryByTestId('selectedProviderButton')).toBeTruthy()
     expect(queryByTestId('nextButton')).toBeTruthy()
   })
+  it('renders header with provider image', () => {
```

Comment:
Maybe a bit overkill, but since the header is relatively customized, could be good to add a test for the back button?

Reply:
Can we also test that analytics events are fired for both presses?